blocks/identity-block/features/facebook-login/default.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { isServerSide } from '@wpmedia/engine-theme-sdk';
+
+// eslint-disable-next-line import/extensions
+import useIdentity from '../../components/Identity.jsx';
```

Comment:
think the `.jsx` can be removed here 

```suggestion
import useIdentity from '../../components/Identity';
```